tests/runfiles/common.run

```diff
@@ -396,6 +396,10 @@ tests = ['zpool_get_001_pos', 'zpool_get_002_pos', 'zpool_get_003_pos',
     'zpool_get_004_neg', 'zpool_get_005_pos']
 tags = ['functional', 'cli_root', 'zpool_get']
 
+[tests/functional/cli_root/zpool_get/vdev]
+tests = ['vdev_get_001_pos']
+tags = ['functional', 'cli_root', 'zpool_get']
```

Comment:
We probably don't want a new test group for this.  I think it'd be best to include this test with the other `zpool_get` tests.

Reply:
so keep `vdev_get_001_pos` but drop the vdev subdirectory, correct?

Reply:
Right.